### PR TITLE
feat: Add extended public key to keystore

### PR DESCRIPTION
**Pull request by TheWaWaR**
So, no need password to use sub-command like `account bip44-addresses`.

* Add `ckb_root` field to keystore json file
* Add `account upgrade` sub-command to update keystore json to latest format

### CODE REVIEW

**Comment by doitian** [comment]
What really does `ckb_root` mean?

**Comment by TheWaWaR** [comment]
> What really does `ckb_root` mean?

`ckb_root` is the ckb token's root path (which is `m/44'/309’/0’`) for pubkey derivation.

Will add a field `ckb_root` to keystore json file to save related information of that path:
- `pubkey`
- `chain code`

With above information, we can do pubkey derivation without require password to decrypt the keystore.

File: ckb-sdk/src/wallet/keystore/mod.rs [hunk 1]
```diff
@@ -157,6 +169,25 @@ impl KeyStore {
...
+                backup_path
+            )));
+        }
+        fs::copy(&filepath, &backup_path)?;
+        let key = self.storage.get_key(hash160, &filepath, password)?;
+        let filename = filepath
+            .file_name()
+            .and_then(OsStr::to_str)
+            .expect("file_name");
+        let _filepath = self.storage.store_key(filename, &key, password)?;
```
Comment (doitian) [inline ckb-sdk/src/wallet/keystore/mod.rs:187]: zeroize the key

Comment (TheWaWaR) [reply ckb-sdk/src/wallet/keystore/mod.rs:187]: `zeroize` logic is located in `impl Drop for MasterPrivKey`

**Pull request merged**